webapp/channels/src/components/post_view/message_attachments/action_button/action_button.tsx

```diff
@@ -26,40 +26,41 @@ export default class ActionButton extends React.PureComponent<Props> {
             good: '#00c100',
             warning: '#dede01',
             danger: theme.errorTextColor,
-            default: theme.centerChannelColor,
+            default: theme.centerChannelBg,
             primary: theme.buttonBg,
             success: theme.onlineIndicator,
```

Comment:
I'm thinking we need to have two different maps here, one for `backgroundColor` and one for `color`